arangosh/Dump/DumpFeature.cpp

```diff
@@ -993,6 +1003,7 @@ Result DumpFeature::storeDumpJson(VPackSlice const& body, std::string const& dbN
     meta.openObject();
     meta.add("database", VPackValue(dbName));
     meta.add("lastTickAtDumpStart", VPackValue(tickString));
+    meta.add("properties", body.get("properties"));
```

Comment:
this may need to make sure that `properties` is an object, and handle it gracefully if not.